src/layerInfos/ZweitausbildungLayerInfo/ZweitausbildungLayerInfo.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
+import { compose } from 'lodash/fp';
+
+const propTypes = {
+  t: PropTypes.func.isRequired,
+  properties: PropTypes.object.isRequired,
+};
+
+const ZweitausbildungLayerInfo = ({ t, properties }) => {
+  const { infos } = properties.get('zweitausbildung');
+  const { title, legend } = infos;
+
+  return (
+    <div className="wkp-zweitausbildung-layer-info">
+      <div>{t(title)}</div>
+      {legend ? (
+        <div className="wkp-zweitausbildung-layer-info-legend">
+          <img
+            src={`${process.env.REACT_APP_STATIC_FILES_URL}/img/layers/zweitausbildung/${legend.image}`}
```

Comment:
I don't think the use of process.env.REACT_APP_STATIC_FILES_URL is a good idea. The urls should come from a property . are we doing this in another layers?

Reply:
or from redux